packages/viewer/src/components/viewer-markup-arrow/viewer-markup-arrow.tsx

```diff
@@ -206,55 +232,59 @@ export class ViewerMarkupArrow {
       const screenEnd = translatePointToScreen(this.end, this.elementBounds);
       const arrowheadPoints = createArrowheadPoints(screenStart, screenEnd);
 
-      return (
-        <Host>
-          <svg class="svg">
-            <defs>
-              <SvgShadow id="arrow-shadow" />
-            </defs>
-            <g filter="url(#arrow-shadow)">
-              <polygon
-                id="arrow-head"
-                class="head"
-                points={arrowheadPointsToPolygonPoints(arrowheadPoints)}
-              />
-              <line
-                id="arrow-line"
-                class="line"
-                x1={screenStart.x}
-                y1={screenStart.y}
-                x2={arrowheadPoints.base.x}
-                y2={arrowheadPoints.base.y}
-              />
-              {this.mode === 'edit' && (
+      if (isValidPointData(screenStart, screenEnd)) {
+        return (
+          <Host>
+            <svg class="svg" onTouchStart={(event) => event.preventDefault()}>
+              <defs>
+                <SvgShadow id="arrow-shadow" />
+              </defs>
+              <g filter="url(#arrow-shadow)">
+                <polygon
+                  id="arrow-head"
+                  class="head"
+                  points={arrowheadPointsToPolygonPoints(arrowheadPoints)}
+                />
                 <line
-                  id="bounding-box-1d-line"
-                  class="bounds-line"
+                  id="arrow-line"
+                  class="line"
                   x1={screenStart.x}
                   y1={screenStart.y}
-                  x2={screenEnd.x}
-                  y2={screenEnd.y}
+                  x2={arrowheadPoints.base.x}
+                  y2={arrowheadPoints.base.y}
                 />
-              )}
-            </g>
-          </svg>
-          {this.mode === 'edit' && (
-            <BoundingBox1d
-              start={screenStart}
-              end={screenEnd}
-              onStartAnchorPointerDown={this.editStartPoint}
-              onCenterAnchorPointerDown={this.editCenterPoint}
-              onEndAnchorPointerDown={this.editEndPoint}
-            />
-          )}
-        </Host>
-      );
+                {this.mode === 'edit' && (
+                  <line
+                    id="bounding-box-1d-line"
+                    class="bounds-line"
+                    x1={screenStart.x}
+                    y1={screenStart.y}
+                    x2={screenEnd.x}
+                    y2={screenEnd.y}
+                  />
+                )}
+              </g>
+            </svg>
+            {this.mode === 'edit' && (
+              <BoundingBox1d
```

Comment:
`BoundingBox2d`?

Reply:
This is just represented as a line along the arrow, so it is technically 1D - though "BoundingBox" does feel like a bit of an incorrect term for that. Nothing immediately comes to mind as a good replacement though